tests/tabular_output/test_output_formatter.py

```diff
@@ -49,3 +69,25 @@ def hello_world(data, headers, **_):
     assert expected == TabularOutputFormatter().format_output(
         data, headers, format_name='ascii', preprocessors=(hello_world,),
         missing_value='hello')
+
+
+def test_format_name_attribute():
+    """Test the the format_name attribute be set and retrieved."""
+    formatter = TabularOutputFormatter(format_name='plain')
+    assert formatter.format_name == 'plain'
+    formatter.format_name = 'simple'
+    assert formatter.format_name == 'simple'
```

Comment:
Isn't this test a little superfluous? If `formatter.format_name = 'simple'` can't be trusted, what can be trusted? :wink: